ipaclient/install/sssd.py

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright (C) 2016  FreeIPA Contributors see COPYING for license
+#
+
+from ipalib.install import service
+from ipalib.install.service import enroll_only
+from ipapython.install.core import group, knob
+
+
+@group
+class SSSDInstallInterface(service.ServiceInstallInterface):
+    description = "SSSD"
+
+    fixed_primary = knob(
+        None,
+        description="Configure sssd to use fixed server as primary IPA server",
+    )
+    fixed_primary = enroll_only(fixed_primary)
+
+    permit = knob(
+        None,
+        description="disable access rules by default, permit all access.",
```

Comment:
I would consider changing it a small doc fix, similar to PEP-8 fixes for Python. But then again I don't mind this in the end, I would probably ACK this even if you rewrote all the help to Pirate English.